validation.h

```diff
@@ -13,37 +10,29 @@ class Validation
 public:
     static bool nameValid(const string &name)
     {
-        return (name.length() >= 5 && name.length() <= 20 && isalpha(name));
+        return (name.length() >= 5 && name.length() <= 20 && stringIsAlphabetic(name));
     }
     static bool passValid(const string &pass)
     {
         return (pass.length() >= 8 && pass.length() <= 20);
     }
-    static bool minBlance(double blance)
+    static bool minBalance(double balance)
     {
-        return (blance >= 1500);
+        return (balance >= 1500);
     }
     static bool minSalary(double salary)
     {
         return (salary >= 5000);
     }
 
 private:
-    static bool isalpha(const string &str)
+    static bool stringIsAlphabetic(string str)
```

Comment:
this function's previous name "isalpha" conflicts with the built in function in cpp so you should have changed it's name